spec/requests/api/v1/contacts/create_spec.rb

```diff
@@ -0,0 +1,68 @@
+RSpec.describe 'POST api/v1/targets', type: :request do
+  let(:user) { create(:user) }
+  let(:contact) { attributes_for(:contact) }
+
+  let(:params) { { contact: contact } }
+
+  let(:headers) { auth_headers(user) }
+
+  subject { post api_v1_contacts_path, params: params, headers: headers, as: :json }
+
+  context 'when the request is correct' do
+    it 'returns a no content response' do
+      subject
+
+      expect(response).to be_no_content
+    end
+
+    it 'creates the contact' do
+      expect {
+        subject
+      }.to change(Contact, :count).by(1)
```

Comment:
maybe you can also add a test that checks that the created contact has the same data as the params